webapp/packages/core-dialogs/src/CommonDialog/CommonDialog/styles.ts

```diff
@@ -48,22 +48,24 @@ export const commonDialogBaseStyle = composes(
     }
     header {
       position: relative;
-      display: flex;
-      flex-direction: column;
+      display: grid;
+      grid-template-columns: 1fr;
       margin-bottom: 24px;
+      &[|icon] {
+        grid-template-columns: max-content 1fr;
+      }
     }
     header-title {
       display: flex;
       align-items: center;
+      justify-content: space-between;
       position: relative;
-      gap: 16px;
       min-height: 24px;
     }
     IconOrImage {
       width: 24px;
       height: 24px;
-      padding-right: 0;
-
+      padding-right: 16px;
```

Comment:
it's a bad idea to use padding inside a square element with a fixed size, seems like you should use margin